src/media/js/helpers_local.js

```diff
@@ -75,6 +75,17 @@ define('helpers_local',
         return array.slice(begin, end);
     };
 
+    filters.fileSize = function(bytes) {
+        var bytes = parseInt(bytes, 10);
+        if (bytes < 1024^2) {
+          return +(bytes/1024).toFixed(2) + ' KB';
```

Comment:
This is syntactically a little confusing. This will help explain:

    >>> (1.1010).toFixed(4);
    "1.1010"

    >>> +(1.1010).toFixed(4);
    1.101

    >>> +(1.1010).toFixed(3);
    1.101

    >>> +(1.1010).toFixed(2);
    1.1

    >>> +(1.1010).toFixed(1);
    1.1

    >>> +(1.1010).toFixed(0);
    1